data/korath/korath missions.txt

```diff
@@ -257,3 +257,78 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old efret rep" = "reputation: Kor Efret"
+		# bring it low enough that capping this will result in war
+		"reputation: Kor Efret" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
+			choice
+				`  "I had not realized they were under your protection. I will leave at once."`
+				`  (Ignore the warning.)`
+					goto ignore
+				`  "I think you should leave. This is my prize and I'll take what I please."`
+					goto refuse
+			`  "Leave then anon, ere we change our mind."`
+				accept
+			label ignore
+			action
+				"reputation: Quarg" <?= -1
+			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`
+				accept
+			label refuse
+			action
+				"reputation: Quarg" <?= -1
+			`  "What impetuous evil is this! Nay, we shall not allow you your plunder; such may not be tolerated." The Quarg open fire before you have a chance to disengage.`
```

Comment:
```suggestion
			`  "What impetuous evil is this? Nay, we shall not allow you your plunder; we shall not abide it." The Quarg open fire before you have a chance to disengage.`
```

First is a question, even as presented. Second I think sits better with the rest than "tolerate," and the switch in subject is a stronger declaration of intent that better matches the ensuing destruction. Alternatively, the start could be simply "What impetuous evil!" as a remark of astonishment instead of a question.

Reply:
I think a question mark would make it sound non-rhetorical.  Questions don't always need question marks; many examples of dated work, not even the Elizabethan I was shooting for here but more, Victorian, say, will end even non-rhetorical questions with periods.  Nor do I agree that the exclamation is undermined by the rhetorical appendage of "is this."

Your second suggestion in this paragraph makes it sound a bit repetitive to me...